docs/sources/enterprise/license/activate-aws-marketplace-license/about-ge-license-through-aws.md

```diff
@@ -8,19 +8,19 @@ weight = 100
 
 # About Grafana Enterprise licenses from AWS Marketplace
 
-AWS Marketplace is a convenient place to buy and manage a Grafana Enterprise license, especially if you are already an AWS customer.
+AWS Marketplace is a convenient place for AWS customers to buy and manage a license for Grafana Enterprise versions 8.3.0 and later.
 
 You can deploy Grafana Enterprise in the following ways:
 
-- You can deploy Grafana Enterprise in AWS ECS or AWS EKS, or,
-- You can deploy Grafana Enterprise on a Grafana Enterprise instance outside AWS.
+- In an instance of AWS ECS or AWS EKS.
```

Comment:
I believe it might be better to change this to:
```suggestion
- Using AWS services like ECS, EKS or EC2.
```
Since any AWS sevice can be used to deploy Grafana using this license.